bundlewrap/repo.py

```diff
@@ -562,7 +562,10 @@ def _build_node_metadata(self, blame=False):
                             # figuring out what changed...
                             input_metadata = deepcopy_metadata(self._node_metadata_partial[node.name])
                         else:
-                            # ...but we can't always do it for performance reasons.
+                            # ...but we can't always do it for performance reasons
+                            # and because we would like to detect if a metaproc
+                            # returned the original dict + {DEFAULTS,OVERWRITE},
+                            # which is illegal.
```

Comment:
I'm a little confused. This new check works with and without the deepcopied metadata, so why the comment?

Reply:
Yeah, brainfart, sorry.